src/clients/FlexbaseClient.Banking.ts

```diff
@@ -12,7 +13,28 @@ interface CreateApplicationResponse extends FlexbaseResponse {
     message?: string;
 }
 
+interface StatementResponse extends FlexbaseResponse {
+  statement?: Statement[] | string;
```

Comment:
Just curious, but why is statement either an array of some object or a string? Is an api limitation? (internal or external)

Reply:
if we make the call to get all the statements, will be an array with the statement data, but if we make the call to get a specific statement, the details will return with an HTML or PDF document